### PR TITLE
Update SECURITY_CONTACTS

### PR DESCRIPTION
**Pull request by cji**
Update the SECURITY_CONTACTS to point to the SRC. 

#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

The existing SECURITY_CONTACTS file was an out of date list of SRC members.

#### Does this PR introduce a user-facing change?
No

```release-note
NONE
```

### CODE REVIEW

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -10,8 +10,4 @@
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
-cjcullen
-joelsmith
-liggitt
-philips
-tallclair
+committee-security-response
```
[the remark this answers is not in the document]
Comment (cji) [reply SECURITY_CONTACTS:13]: good call, will update!

[the remark this answers is not in the document]
Comment (cji) [reply SECURITY_CONTACTS:15]: @nikhita added a link above to our contact information, can you please take another look?

**Review by nikhita — APPROVED** [review]
/lgtm 

Thanks @cji!

/assign @dims 
for approval

**Pull request merged**